PUBLISHING.md

```diff
@@ -22,10 +23,29 @@ This will:
 - Push this tag to the remote
 - Trigger the new version publish
 
-## Rollback a new Release
+You can use the "Release latest-rc" manual github action to publish the last commit on main as the `latest-rc` version
 
+## Update tag version
+
+```sh
+$ yarn npm tag add sfdx-git-delta@<version-source> <version-target> 
+```
+
+It will set the version-source release channel to the version-target.
+
+Per example if you need to rollback `latest` to `v5.0.0`:
```

Comment:
```suggestion
**Rollback**: 
Use this command only if something is wrong with the current `latest` version, and you need to roll it back to a previous version (to `v5.0.0` in this example).
```

Reply:
Suggestion: I would maybe put this scenario at the bottom of this **Update tag version** chapter, as this command is not needed as part of the "happy flow".